simulator/simulator.py

```diff
@@ -0,0 +1,65 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+import datetime
+import time
+import json
+import secrets
+from pprint import pprint
+
+KEY_PATH = './firebase.json'
+COORDINATES_PATH = 'coordinates.json'
+DATABASE_REFERENCE_PATH = 'datas/'
+DATABASE_URL = 'https://smartspace-84906-default-rtdb.firebaseio.com/'
+
+random = secrets.SystemRandom()
+
+cred = credentials.Certificate(KEY_PATH)
+firebase_admin.initialize_app(cred, {
+        'databaseURL': DATABASE_URL
+})
+
+with open(COORDINATES_PATH) as json_file:
+    coordinates_data = json.load(json_file)
+
+used_coordinates_count = 0
+
+def insert_data(data):
+    # timestamp = str(int(time.time() * 1000000))
+    ref = db.reference(DATABASE_REFERENCE_PATH)
+    ref.set(data)
+    pprint('Send data to db: {}'.format(data))
+
+def create_data():
+    global used_coordinates_count
+    try:
+        coordinate = coordinates_data[used_coordinates_count]
+        if used_coordinates_count == len(coordinates_data):
+            used_coordinates_count = 0
+        else:
+            used_coordinates_count += 1
+    except:
```

Comment:
Suggest adding a clearly exception type here.
```suggestion
    except XXXXException:
```

Reply:
Addressed at https://github.com/ricky9667/smart_space/pull/14/commits/cb7e1d9479cfe848764af0f25c12618f78c85ac2.